packages/web/utils/mathHelper.ts

```diff
@@ -0,0 +1,5 @@
+// true if the number is a power of 2
+export const isPow2 = (int: number): boolean => (
+  // eslint-disable-next-line no-bitwise
```

Comment:
let's just turn `no-bitwise` off in the lint config

Reply:
Done.